### PR TITLE
New variables structure

### PR DESCRIPTION
**Pull request by dokshina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,16 +23,17 @@ This role can deploy and configure applications packed in RPM using
   * [Cartridge authorization](#cartridge-authorization)
   * [Application configuration](#application-configuration)
 
-## Requirements
```
Comment (artur-barsegyan) [inline README.md:23]: Do you want to move this section to the README?

File: README.md [hunk 2]
```diff
@@ -52,61 +56,70 @@ To deploy an application and set up this topology:
...
+    cartridge_package_path: ./myapp-1.0.0-0.rpm
+    cartridge_cluster_cookie: secret-cookie
+    cartridge_defaults:
+      log_level: 1
+
+  children:
+    # group instances by machines
+    host1:
+      vars:
+        # first machine addres and connection opts
```
Comment (artur-barsegyan) [inline README.md]: typo: address

File: README.md [hunk 3]
```diff
@@ -52,61 +56,70 @@ To deploy an application and set up this topology:
...
+      hosts:  # instances to be started on this host
+        storage-1:
+          config:
+            advertise_uri: '172.19.0.3:3301'
+            http_port: 8091
+
+    # group instances by replicasets
+    storage_1_replicaset:  # replicaset storage-1
+      hosts:  # instances
+        storage-1:
```
Comment (artur-barsegyan) [inline README.md:103]: Why it uses dictionary instead of list of strings?

File: README.md (hunk 1 above)
Comment (artur-barsegyan) [reply README.md:23]: ???

File: README.md [hunk 4]
```diff
@@ -143,8 +152,76 @@ Configuration format is described in detail in the
...
+  * configure authorization (if `cartridge_auth` is defined);
+  * patch application clusterwide config (if `cartridge_app_config` is defined);
+  * bootstrap Vshard (if `cartridge_bootstrap_vshard` is defined);
+  * manage cartridge failover (if `cartridge_failover` is defined).
+
+**Note**, that `cartridge-config` tasks would be skipped if no one of `cartridge_auth`, `cartridge_app_config`, `cartridge_bootstrap_vshard` and `cartridge_failover` variables is specified.
+
+### Example scenario
+
+Using `--tags` and `--limits` options you can manage cluster different ways:
```
Comment (artur-barsegyan) [inline README.md]: Create one example without `--tags`. 
First reading of this section was unobvious.

File: README.md (hunk 3 above)
Comment (dokshina) [reply README.md:103]: Because of Ansible specific

**Pull request merged**